### DISCUSSION TITLE
How to hide the title bars on tabbed/stacked mode?

### DISCUSSION THREAD

**Discussion by falcie1337** [i3/i3#5836] [Q&A] [answered]
I've just installed i3 today and am configuring it in order to make it look a little bit better for my taste, and one thing i don't like are these title bars, i already searched through the internet yet i've found nothing about how to actually hide them. I've managed to hide the windows in tiling mode, but on tabbed/stacked mode it will still appear, which is a bummer for me since i'm using polybar and it already shows the window title there, so it is just a waste of space. I've seen a video about The Linux Cast on i3 tabs and he actually managed to hide his tabbed mode title bars (the title bars on tabbed mode get down to 1 pixel, so it only highlights the tab he is in instead of showing the entire thing) however, he doesn't show how he did it in his config file and i couldn't find anything helpful online. If someone knows how to do it, i would greatly appreciate it!

Link to his video here, so you can see more clearly what i'm talking about: https://youtu.be/sHWYr63mxpg&t=285

P.S.: I'm also only using the standard tabbed i3 layout instead of the tabbed windows he is using, just for clarification (i don't really know if there would be a difference)

**Comment by okraits** [comment]
In tabbed and stacked mode, titlebars can't be hidden completely. You can set a font size of 0 so the title bar height is very small.

**Reply by falcie1337** [reply]
That's exactly what i did after posting this here and it indeed worked. Thank you for your time answering to my post anyways, cheers! :D

**Comment by mikeredev** [comment, marked as answer]
his dotfiles are [here](https://gitlab.com/thelinuxcast/my-dots/-/tree/master/i3?ref_type=heads)

I believe it's the combination of `font pango:monospace 0` and `default_border pixel 3` that achieves that effect. or something along those lines. I don't think that's a great setup though, as if you have multiple tabs/stacks open it's going to be nearly impossible to navigate through them.

**Reply by falcie1337** [reply]
I already figured that out, i don't use many tabs, so for me it's alright. I like using them but i don't open more than 3-4 tabs and i can very clearly navigate through them with the slim title-bar as an indicator of which tab i'm in, so tbh it's fine. Thank you for your comment anyways. :)
